### DISCUSSION TITLE
We need the total of Failed users per phase

### DISCUSSION THREAD

**Discussion by sruthii38** [artilleryio/artillery#2573] [Artillery]
Artillery is providing the total Failed users in the Metric and also listing it as a graph(failed count vs time) in Report. But we are looking to get the failed user count per phase. It is really time consuming to add up the numbers for each phase to present to the dev. Is there anyway I can get this metrics reported.

Version info: 2.0.0-38

I expected to see this happen: Total failed users/ phase of the test.

Instead, this happened: we Only have total failed users for the whole test.

Files being used: Artillery report

**Comment by hassy** [comment]
thanks for the feature request @sruthii38! We'll look at adding it at some point, but don't have an ETA yet. In the meantime, you can create a JSON report for each test run with the `--output` flag ([docs here](https://www.artillery.io/docs/reference/cli/run#--output---create-a-json-report)) and get the count of `vusers.failed` in each phase.

**Reply by hassy** [reply]
(moving to Discussions)
